src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,271 @@
+                "██║░░██╗░██║░░██║██║░░██║██║░░██║██████╦╝░╚████╔╝░█████╗░░\n" +
+                "██║░░╚██╗██║░░██║██║░░██║██║░░██║██╔══██╗░░╚██╔╝░░██╔══╝░░\n" +
+                "╚██████╔╝╚█████╔╝╚█████╔╝██████╔╝██████╦╝░░░██║░░░███████╗\n" +
+                "░╚═════╝░░╚════╝░░╚════╝░╚═════╝░╚═════╝░░░░╚═╝░░░╚══════╝\n" +
+                "Hope to see you again soon!");
+    }
+
+    public static void welcome() {
+        System.out.println("\n" +
+                "██╗░░██╗███████╗██╗░░░░░██╗░░░░░░█████╗░\n" +
+                "██║░░██║██╔════╝██║░░░░░██║░░░░░██╔══██╗\n" +
+                "███████║█████╗░░██║░░░░░██║░░░░░██║░░██║\n" +
+                "██╔══██║██╔══╝░░██║░░░░░██║░░░░░██║░░██║\n" +
+                "██║░░██║███████╗███████╗███████╗╚█████╔╝\n" +
+                "╚═╝░░╚═╝╚══════╝╚══════╝╚══════╝░╚════╝░\n" +
+                "from\n" + "\n" +
+                "██████╗░██╗░░░██╗██╗░░██╗███████╗\n" +
+                "██╔══██╗██║░░░██║██║░██╔╝██╔════╝\n" +
+                "██║░░██║██║░░░██║█████═╝░█████╗░░\n" +
+                "██║░░██║██║░░░██║██╔═██╗░██╔══╝░░\n" +
+                "██████╔╝╚██████╔╝██║░╚██╗███████╗\n" +
+                "╚═════╝░░╚═════╝░╚═╝░░╚═╝╚══════╝\n");
+        System.out.println("Hello! I'm Duke\nWhat can I do for you?");
+        System.out.println("_________________________________________________________________________");
+    }
+
+    public static void printInvalidMessage(String type) {
+        System.out.println("\uD83D\uDE1E OOPS! " + type + " is not valid, please try again!");
+    }
+
+    public static void validateInput (String input, String type) throws DukeException {
+        try {
+            if (type.equals("done") && (!(input.strip()).matches("[-+]?\\\\d*\\\\.?\\\\d+\n"))) {
+                throw new DukeException();
+            } else if (type.equals("todo") && (input.strip().length() <= 0)) {
+                throw new DukeException();
+            } else {
+                if (!input.contains("/")) {
+                    if (type.equals("deadline")) {
+                        throw new DukeException();
+                    } else if (type.equals("event")){
+                        throw new DukeException ();
+                    }
+                }
+            }
+        } catch (IndexOutOfBoundsException e) {
+            printInvalidMessage(type);
+        }
+    }
+
+    public static void showList() {
+        if (tasks.size() > 0) {
+            System.out.println("Here are the tasks in your list:");
+            for (int i = 0; i < tasks.size(); i++) {
+                Task currentTask = tasks.get(i);
+                System.out.print(i+1 + ". " + currentTask.getTaskType() + currentTask.getStatusIcon() + " "
+                    + currentTask.getDescription());
+                if (currentTask.getTaskType() == "[D]")
+                    System.out.print(" (by: " + currentTask.getDateTime() + ")");
```

Comment:
Do remember to add braces to `if` statements even if it contains only 1 statement to be executed.